comms/question_test.go

```diff
@@ -0,0 +1,33 @@
+package comms
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestQuestion(t *testing.T) {
+	tests := []struct {
+		desc     string
+		given    string
+		fallback string
+		expected string
+	}{
+		{"records response", "hello\n", "", "hello"},
```

Comment:
Oh. I meant to add it to the assertion for context. What is missing from the context in terms of detail?